landing-pages/site/content/en/blog/documenting-using-local-development-environments.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Documenting using local development environment"
+linkTitle: "Documenting using local development environment"
+author: "Elena Fedotova"
+github: "efedotova"
+linkedin: "elena-fedotova-039294110"
+description: "The story behind documenting local development environment of Apache Airflow"
+tags: ["Development"]
+date: 2019-11-22
+---
+
+## Documenting local development environment of Apache Airflow
+
+From Sept to November, 2019 I have been participating in a wonderful initiative, [Google Season of Docs](https://developers.google.com/season-of-docs).
+
+I had a pleasure to contribute to the Apache Airflow open source project as a technical writer.
+My initial assignment was an extension to the github-based Contribution guide.
+
+From the very first days I have been pretty closely involved into inter-project communications
+via emails/slack and had regular 1:1s with my mentor, Jarek Potiuk.
+
+I got infected with Jarek’s enthusiasm to ease the on-boarding experience for
+Airflow contributors. I do share this strategy and did my best to improve the structure,
+language and DX. As a result, Jarek and I extended the current contributor’s docs and
+ended up with the Contributing guide navigating the users through the project
```

Comment:
```
ended up with [the Contributing guide](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst) navigating the users through the project
```

Reply:
Yes. In this sentence it makes sense for the link to be at the beginning because that's where the subject of the sentence is logically. In the  GSOD sentence "Me" (Elena) is the subject and GSOD is the object (https://www.quickanddirtytips.com/education/grammar/subject-and-object).